telemetry/definitions/commonDefinitions.json

```diff
@@ -214,6 +225,11 @@
             "name": "insightsQueryStringType",
             "allowedValues": ["insights", "searchTerm"],
             "description": "User-specified search string type while starting an insights query"
+        },
+        {
+            "name": "ecsLaunchType",
+            "allowedValues": ["ec2", "fargate"],
+            "description": "Type of ECS Task"
```

Comment:
"Compute environment for ECS task"?

Reply:
Revised to "Infrastructure type used by ECS tasks and services" based on https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html